src/WinRT.Runtime/Projections/INotifyPropertyChanged.net5.cs

```diff
@@ -2,32 +2,34 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Reflection;
 using System.Runtime.InteropServices;
 using WinRT;
+using WinRT.Interop;
 
 namespace ABI.System.ComponentModel
-{
+{
 #if EMBED
     internal
 #else
     public
 #endif
     static class INotifyPropertyChangedMethods
-    {
-        public static global::System.Guid IID { get; } = new Guid(new global::System.ReadOnlySpan<byte>(new byte[] { 0x01, 0x76, 0xB1, 0x90, 0x65, 0xB0, 0x6E, 0x58, 0x83, 0xD9, 0x9A, 0xDC, 0x3A, 0x69, 0x52, 0x84 }));
+    {
+        public static global::System.Guid IID { get; } = GuidGenerator.GetWuxMuxIID(typeof(INotifyPropertyChanged).GetCustomAttribute<WuxMuxProjectedTypeAttribute>());
```

Comment:
Not sure why the comment here was resolved, I'm still seeing the attribute lookup? Or do you just plan on addressing all of these later when you get around to rebasing this PR? I won't leave other comments about this for now 😄